backend/src/api/transactions/transaction.controller.js

```diff
@@ -0,0 +1,28 @@
+const catchAsync = require("../../utils.js/catchAsync");
+const TransactionModel = require('./transaction.model');
+
+
+
+const addTransactionController = catchAsync(async (req, res, next) => {
+    const { body } = req;
+    const newTransaction = await TransactionModel.addTransaction(body);
+    res.status(201).json({
+        status: 'success',
+        transaction: newTransaction,
+    });
+});
+
+const getTransactionController = catchAsync(async (req, res) => {
```

Comment:
`getTransactions` in this case. Don't know why we're need to add `Controller` suffix